source/_integrations/nice_go.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: Nice G.O.
+description: Control Nice G.O. garage doors
+ha_release: '2024.9'
+ha_category:
+  - Cover
+  - Light
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@IceBotYT'
+ha_domain: nice_go
+ha_platforms:
+  - cover
+  - diagnostics
+  - light
+ha_integration_type: integration
+---
+
+The **Nice G.O.** {% term integration %} lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
+
```

Comment:
Are there any prerequisites? Any steps I need to do before I can get started with the integration? Install an app, have login credentials, etc?

Reply:
Prerequisite is to have a working account with Nice G.O.